ScoutSuite/providers/gcp/facade/facade.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from googleapiclient import discovery
+from ScoutSuite.providers.gcp.utils import MemoryCache
+
+class GCPFacade:
+    def __init__(self):
+        self._resourcemanager_client = discovery.build('cloudresourcemanager', 'v1', cache_discovery=False, cache=MemoryCache())
```

Comment:
It seems like this method potentially does heavy stuff (http requests). I don't think it's a good idea to do this in a constructor. See the [method's implementation](https://github.com/googleapis/google-api-python-client/blob/84cbc07ca4349b2b52f210c8c6f861dd1b178d21/googleapiclient/discovery.py#L170).

Reply:
Good point!